kratos/processes/from_json_check_result_process.cpp

```diff
@@ -871,15 +870,14 @@ FromJSONCheckResultProcess::ElementsArrayType& FromJSONCheckResultProcess::GetEl
             # pragma omp for schedule(guided, 512) nowait
             for (int i = 0; i < number_of_elements; ++i) {
                 auto it_elem = it_elem_begin + i;
-                if (CheckFlag(*it_elem, pFlag)) elements_buffer.push_back(*(it_elem.base()));
+                if (CheckFlag(*it_elem, pFlag)) elements_buffer.insert(elements_buffer.end(), *(it_elem.base()));
```

Comment:
inefficient. this should be unordered so thar you can push back on it and sort it at the end